pkg/services/grafana-apiserver/service.go

```diff
@@ -290,7 +300,25 @@ func (s *service) start(ctx context.Context) error {
 		return genericapiserver.DefaultBuildHandlerChain(requestHandler, c)
 	}
 
+	xxx := k8sversion.Get()
+	jjj, _ := json.MarshalIndent(xxx, "", "  ")
+	fmt.Printf(">>> %s\n%v\n", string(jjj), setting.BuildStamp)
+
 	serverConfig.TracerProvider = s.tracing.GetTracerProvider()
+	before, after, _ := strings.Cut(setting.BuildVersion, ".")
+	serverConfig.Version = &version.Info{
+		Major:        before,
+		Minor:        after,
+		GoVersion:    goruntime.Version(),
+		Platform:     fmt.Sprintf("%s/%s", goruntime.GOOS, goruntime.GOARCH),
+		Compiler:     goruntime.Compiler,
+		GitTreeState: setting.BuildBranch,
+		GitCommit:    setting.BuildCommit,
+		BuildDate:    time.Unix(setting.BuildStamp, 0).UTC().Format(time.DateTime),
+
+		// This is used by kubectl to check compatibility.
+		GitVersion: "v1.28.3", // ???? how do we get this programmatically
```

Comment:
Any better ideas how to get this programmatically?

I tried both [k8s.io/client-go/pkg/version](https://github.com/kubernetes/kubernetes/blob/v1.28.3/staging/src/k8s.io/client-go/pkg/version/version.go) and [k8s.io/component-base/version](https://github.com/kubernetes/kubernetes/blob/v1.28.3/staging/src/k8s.io/component-base/version/version.go)
```
xxx := k8sversion.Get()
jjj, _ := json.MarshalIndent(xxx, "", "  ")
```

But neither return 
```
{
  "major": "",
  "minor": "",
  "gitVersion": "v0.0.0-master+$Format:%H$",
  "gitCommit": "$Format:%H$",
  "gitTreeState": "",
  "buildDate": "1970-01-01T00:00:00Z",
  "goVersion": "go1.21.4",
  "compiler": "gc",
  "platform": "darwin/arm64"
}
```

Reply:
Our response is now:
```
I1113 15:03:40.906536    6280 request.go:1212] Response Body: {
  "major": "10",
  "minor": "3.0-pre",
  "gitVersion": "v1.28.3",
  "gitCommit": "1c8b2ecd65",
  "gitTreeState": "k8s-add-version",
  "buildDate": "2023-11-13 23:02:35",
  "goVersion": "go1.21.4",
  "compiler": "gc",
  "platform": "darwin/arm64"
}
Client Version: v1.28.3
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.28.3
```